Applications/CLI/Tests.cmake

```diff
@@ -18,6 +18,18 @@ if(NOT OGS_USE_MPI)
         )
 
         AddTest(
+            NAME GroundWaterFlowProcess_cube_1x1x1_${mesh_size}_Newton
```

Comment:
The other tests of the linear GWFlow use Picard, which also doesn't make sense in a way.
It is tested that the central differences assembly works.